src/chat.rs

```diff
@@ -2795,6 +2883,75 @@ pub(crate) async fn add_info_msg(context: &Context, chat_id: ChatId, text: impl
     });
 }
 
+/// Get autodelete timer value in seconds.
+pub async fn get_autodelete_timer(context: &Context, chat_id: ChatId) -> u32 {
```

Comment:
Any reason to not use something stronger typed here, like `std::time::Duration`?

Reply:
Fixed, added `ephemeral::Timer`